Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/FeedRecordedFuture.py

```diff
@@ -599,15 +614,21 @@ def main():  # pragma: no cover
         'rf-feed-get-risk-rules': get_risk_rules_command
     }
     try:
-        if demisto.command() == 'fetch-indicators':
-            if client.risk_rule:
-                for risk_rule in client.risk_rule:
-                    demisto.debug(f'RF: Fetching indicators for {risk_rule=}')
-                    fetch_and_create_indicators(client, risk_rule)
-            else:  # there are no risk rules
-                demisto.debug('RF: Fetching indicators without risk rules')
-                fetch_and_create_indicators(client)
-
+        if demisto.command() == "long-running-execution":
+            while True:
+                try:
+                    run_fetch_indicator(client)
+                except Exception as e:
+                    demisto.error(f'RF: Exception in long-running-execution: {str(e)}')
+                finally:
+                    sleep_time_in_minutes = interval * MINUTES
+                    demisto.debug(f"Sleeping for {sleep_time_in_minutes}")
+                    sleep(sleep_time_in_minutes)
```

Comment:
isn't the arg of sleep is time in seconds?

Reply:
The interval parameter from XSOAR is in minutes but the sleep is in seconds so we need to convert it.